### PR TITLE
Add Bluetooth.getDevices

### PR DESCRIPTION
**Pull request by hamishwillee**
A checklist to help your pull request get merged faster:
- [x] Summarize your changes
- [x] Data: link to resources that verify support information (such as browser's docs, changelogs, source control, bug trackers, and tests)
- [x] Data: if you tested something, describe how you tested with details like browser and version
- [x] Review the results of the linter and fix problems reported (If you need help, please ask in a comment!)
- [ ] Link to related issues or pull requests, if any

This adds documentation for the new Bluetooth.getDevices() which was recently added to chromium (build 85) under a flag. Some docs on this here:
- https://docs.google.com/document/d/1RF4D-60cQJWR1LoQeLBxxigrxJwYS8nLOE0qWmBF1eo/edit
- https://googlechrome.github.io/samples/web-bluetooth/get-devices.html
- https://www.chromestatus.com/feature/4797798639730688

I tested this for Chrome, Edge, Opera for desktop on Windows. I rolled out to equivalent versions for some of the others. There are a couple of questions inline too.

FYI, this is one of the missing entries listed in https://github.com/mdn/browser-compat-data/issues/7585

PS When this is done will need to add compatibility table to https://developer.mozilla.org/en-US/docs/Web/API/Bluetooth/getDevices

### CODE REVIEW

File: api/Bluetooth.json [hunk 1]
```diff
@@ -203,6 +203,89 @@
...
+                  "type": "preference",
+                  "name": "#enable-experimental-web-platform-features",
+                  "value_to_set": "enabled"
+                }
+              ]
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": false,
```
Comment (hamishwillee) [inline api/Bluetooth.json:284]: @ddbeck The parent `Bluetooth` and friends had `true` for this. However the spec https://webbluetoothcg.github.io/web-bluetooth/ seems to indicate it is not - " It is not a W3C Standard nor is it on the W3C Standards Track. " - perhaps I misunderstand

File: api/Bluetooth.json [hunk 2]
```diff
@@ -203,6 +203,89 @@
...
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            },
+            "samsunginternet_android": {
+              "version_added": false
+            },
+            "webview_android": {
+              "version_added": "85",
```
Comment (hamishwillee) [inline api/Bluetooth.json:272]: Note, this one I did not test (don't know how to) and whether the flag is relevant. Thoughts @ddbeck ?

File: api/Bluetooth.json [hunk 3]
```diff
@@ -203,6 +203,89 @@
...
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "#enable-experimental-web-platform-features",
+                  "value_to_set": "enabled"
+                }
+              ]
+            },
+            "chrome_android": {
+              "version_added": "85"
```
Comment (ddbeck) [inline api/Bluetooth.json:221]: Is this actually not behind a preference on Android, but is behind a preference on desktop?

File: api/Bluetooth.json (hunk 2 above)
Comment (ddbeck) [reply api/Bluetooth.json:272]: WebView doesn't support any of Chrome's flags. So if this is behind a flag on Chrome for Android, then it's likely to be `false` for WebView (this is also the cause of the linter error—the parent feature is `false` for WebView).

File: api/Bluetooth.json (hunk 1 above)
Comment (ddbeck) [reply api/Bluetooth.json:284]: Historically, there was some uncertainty for a bit about whether we should accept WICG proposals as standards track or not. But I think we have consensus now that we should take specs at their word when they say "It is not a W3C Standard nor is it on the W3C Standards Track".

Want to open a follow-up PR or issue to fix these values? 😄

Comment (hamishwillee) [reply api/Bluetooth.json:284]: @ddbeck I hate you :-) . Done in #8012

File: api/Bluetooth.json (hunk 3 above)
Comment (hamishwillee) [reply api/Bluetooth.json:221]: @ddbeck This IS behind a preference on desktop (or at least the ones I tested - chrome, opera, edge).  I _do not know_ whether it is behind a preference on android (links don't say). How do we find out?

File: api/Bluetooth.json (hunk 2 above)
Comment (hamishwillee) [reply api/Bluetooth.json]: OK, I have made this false.

File: api/Bluetooth.json (hunk 3 above)
Comment (ddbeck) [reply api/Bluetooth.json:221]: I ended up [testing this](https://codepen.io/ddbeck/pen/ZEpyvLa?editors=1111) on my phone. In Chrome 87, this is still behind the flag.

File: api/Bluetooth.json [hunk 4]
```diff
@@ -203,6 +203,82 @@
...
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "#enable-experimental-web-platform-features",
+                  "value_to_set": "enabled"
+                }
+              ]
+            },
+            "opera_android": {
+              "version_added": "61",
```
Comment (ddbeck) [inline api/Bluetooth.json:253]: Sorry I missed this previously. We ought to set this to `false`, actually. Opera for Android doesn't support flags, last I checked. We really ought to do some enforcement of this.

**Review by ddbeck — CHANGES_REQUESTED** [review]

File: api/Bluetooth.json (hunk 3 above)
Comment (hamishwillee) [reply api/Bluetooth.json]: Thanks! Fixed

File: api/Bluetooth.json (hunk 4 above)
Comment (hamishwillee) [reply api/Bluetooth.json]: Fixed (set to false)

**Comment by hamishwillee** [comment]
Thanks @ddbeck , I think this addresses your comments now

**Review by ddbeck — APPROVED** [review]
Looks great. Thank you! 🎉

**Pull request merged**